tests/integration/components/bs-dropdown-test.js

```diff
@@ -137,6 +137,32 @@ module('Integration | Component | bs-dropdown', function(hooks) {
     assert.dom(dropdownVisibilityElementSelector()).hasClass(openClass(), 'Dropdown is open');
   });
 
+  test('clicking dropdown menu when closeOnMenuClick is false will not close it, even after opening the menu again', async function (assert) {
```

Comment:
Looks good. Did you check that this was failing without the change in the component?